WinUIGallery/Samples/ControlPages/Design/TypographyPage.xaml

```diff
@@ -212,6 +215,21 @@
                             SizeLineHeight="14/20 epx"
                             VariableFont="Text, SemiBold"
                             Weight="600" />
+                        <controls:TypographyControl
+                            Example="Body Large"
```

Comment:
Is there a use-case for these two?

Reply:
Yeah, especially whenever a Subtitle would be too large - and BodyStrong to small - this gives the developer a bit more flexibility.

Reply:
Is there a visual example that can be added to the image?